ports/mimxrt/README.md

```diff
@@ -29,3 +29,36 @@ Known issues:
 TODO:
   - More peripherals (Counter, I2S, CAN, etc)
   - More Python options
+
+## Building
+
+``` bash
+cd ports/mimxrt
+```
+
+Make sure you have the appropriate c library installed
```

Comment:
Maybe add the compiler package too? (It seems unlikely you'd have the compiler but not newlib?)